### PR TITLE
Use vets json schema fixtures for pension cypress tests

### PR DESCRIPTION
**Pull request by scottsdevelopment**
## Summary

- Point cypress test data to the new vets json schema examples from the package.

## Related issue(s)

- department-of-veterans-affairs/va.gov-team#75946

## Testing done

- All cypress tests pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -318,7 +318,7 @@
     "url-search-params-polyfill": "^8.1.1",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#39dd490a3031bbb28ec30acdb1efa131122a77c8"
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#a07ab99bc6d6f5896f7315ad21de5507737e1401"
```
Comment (mjknight50) [inline package.json:321]: Would it make sense to move away from git commit toward version number?

**Review by danlim715 — DISMISSED** [review]

**Comment by Andrew565** [comment]
PR has failing required tests, opting not to review at this time. - Tier 2/FE COP

**Review by wayne-weibel — CHANGES_REQUESTED** [review]
Am seeing the below error when running locally
`yarn cy:run --spec src/applications/pensions/tests/e2e/pensions.cypress.spec.js`

image

**Review by mjknight50 — DISMISSED** [review]
I ran some tests and it looked good.

**Review by wayne-weibel — APPROVED** [review]
If all other cypress tests are still passing this will work for now

**Comment by wayne-weibel** [comment]
@Andrew565 all of the required tests (should) be passing now.

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**